src/input/input.styles.scss

```diff
@@ -148,7 +160,13 @@
   }
 }
 
+.error ~ .msg {
+  opacity: 1;
```

Comment:
We quickly speak about it.
Opacity transition are not good for mobile.

Reply:
it's the same problem for label, but, for the moment, i've no idea for sexy UI whithout this?
Translation ?

Reply:
Let it this way right now, I don't know how to do this fine.

 - Maybe adding a `transform: scale(1)` would do the trick (it force the navigator to use GPU)
 - This is worth doing some benchmark one day